web/client/reducers/mapInfo.js

```diff
@@ -92,9 +99,13 @@ function mapInfo(state = initState, action) {
         });
     }
     case CHANGE_MAPINFO_FORMAT: {
-        return assign({}, state, {
-            infoFormat: action.infoFormat
-        });
+        return {...state,
+            infoFormat: action.infoFormat,
```

Comment:
I would remove this one and use only the one in configuration, updating the related selectors. I would like to avoid duplicating stuff in the state

Reply:
nice catch. going to fix it